README.md

```diff
@@ -95,5 +85,8 @@ docker run cckp-toolkit-workflow https://github.com/mc2-center/cckp-toolkit-work
 **Subset of tools to test**: Any from [this list](https://cancercomplexity.synapse.org/Explore/Tools) with a GitHub repository.
 
 ## Notes
-
-Ensure Git is installed on your system as the workflow uses `git clone` to clone the repository. The workflow assumes the repository is public or accessible with the provided URL.
+- Ensure Git is installed on your system as the workflow uses `git clone` to clone the repository. The workflow assumes the repository is public or accessible with the provided URL.
+- The entire workflow is containerized, so you only need Docker installed.
```

Comment:
Change up to only need nextflow & docker

Reply:
It would be very easy to add conda envs for each process as well adding more deployment options